python/cuml/metrics/trustworthiness.pyx

```diff
@@ -102,7 +102,7 @@ def trustworthiness(X, X_embedded, handle=None, n_neighbors=5,
 
     cdef cumlHandle* handle_ = <cumlHandle*>0
     if handle is None:
-        handle_ = <cumlHandle*><size_t>(new cumlHandle())
+        handle_ = <cumlHandle*><size_t>(new cumlHandle(3))
```

Comment:
Wouldn't we want to set this to the default as well? Trustworthiness isn't making use of this property.

Reply:
Done.